Calculator.gkemeza/CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Newtonsoft.Json;
+
+namespace CalculatorLibrary
+{
+    public class Calculator
+    {
+        JsonWriter writer;
+        public List<string> calculations = new List<string>();
+
+        public Calculator()
+        {
+            StreamWriter logFile = File.CreateText("calculatorlog.json");
+            logFile.AutoFlush = true;
+            writer = new JsonTextWriter(logFile);
+            writer.Formatting = Formatting.Indented;
+            writer.WriteStartObject();
+            writer.WritePropertyName("Operations");
+            writer.WriteStartArray();
+        }
+
+        public double DoOperation(double num1, double num2, string op)
+        {
+            double result = double.NaN; // Default value is "not-a-number" if an operation, such as division, could result in an error.
+            writer.WriteStartObject();
+            writer.WritePropertyName("Operand1");
+            writer.WriteValue(num1);
+            writer.WritePropertyName("Operand2");
+            writer.WriteValue(num2);
+            writer.WritePropertyName("Operation");
+
+            switch (op)
+            {
```

Comment:
For the code itself i have 2 suggestions for future projects and this one if you want to fix it in the future.

- If you use the program a lot, the console becomes too crowded. Think of ways you could clear the console after certain operation to keep the ui clean 
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Calculator/assets/12613508/40b188fd-73c0-4244-bf44-10a45fa30649)

- Your switch statement became a bit too big. This is not a problem in itself, but try to think of ways you could make it smaller.